src/IO/SeekableStreamingReadBuffer.h

```diff
@@ -0,0 +1,68 @@
+#pragma once
+
+#include <IO/ReadBufferFromFileBase.h>
+
+
+namespace DB
+{
+
+class SeekableStreamingReadBuffer : public ReadBufferFromFileBase
```

Comment:
Missing comments.

Reply:
I guess with new name it doesn't need comments

Reply:
+